app/styles/image-upload.css

```diff
@@ -5,7 +5,7 @@
 }
 
 .drop-area {
-  border: 2px dashed #ccc;
+  border: 2px dashed var(--drop-area-border);
```

Comment:
Refer to this comment
https://github.com/Real-Dev-Squad/website-my/pull/321/files#r1030068003